docs/noise/_index.yaml

```diff
@@ -0,0 +1,37 @@
+book_path: /cirq/_book.yaml
+project_path: /cirq/_project.yaml
+title: Noise management for running circuits
+landing_page:
+  nav: left
+  rows:
+    - heading: Manage noise when running circuits
+      description: Running circuits on real quantum devices means dealing with the noise those devices introduce to the computation. Cirq provides different ways to managing that noise, to improve the reliability of a circuit when run on hardware.
+    - heading: Characterization and compensation
+      description: By first characterizing how much and what kinds of error a device is exhibiting, compensation can be used to improve the reliability of the next run on the device or interpret the results of a previous run more accurately.
+      items:
+        - heading: "Calibration: Overview and API"
+          description: Client-side characterization and compensation (aka calibration) procedures.
```

Comment:
I think this page is about the Calibration API, which is server-side characterization and compensation, not client-side.